stratum/tools/stratum-replay/README.md

```diff
@@ -0,0 +1,28 @@
+<!--
+Copyright 2020-present Open Networking Foundation
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+P4Runtime write request replay tool
+====
+
+This tool replay P4Runtime write requests to a Stratum device with a given
+Stratum P4Runtime write request log.
+
+# Usage:
+
+`stratum-replay [options] [p4runtime write log file]`
+
+# Options:
+
+```
+-device_id: The device ID (default: 1)
+-election_id: Election ID (high,low) for abstraction update (default: "0,1")
+-grpc_addr: Stratum gRPC address (default: "127.0.0.1:9339")
+-p4info: The P4Info file (default: "p4info.pb.txt")
```

Comment:
I'm surprised to see the p4info as a command-line argument, shouldn't this be included in the `pipeline.pb.bin` file? If it's not, then I don't understand the purpose of `pipeline.pb.bin`.

Reply:
The `pipeline.pb.bin` is the one generated by the pipeline builder tool. Which is the `p4_device_config` and it shouldn't include the p4info

Reply:
Or should we use the pipeline config that stored in the stratum container?(the one created by stratum when it receives the pipeline config)

Reply:
> Or should we use the pipeline config that stored in the stratum container?(the one created by stratum when it receives the pipeline config)

This might be easier for the user since they don't need to get the pipeline config from the control plane

Reply:
Yes. I thought `pipeline.pb.bin` was the one generated by Stratum, I guess it's easy to get confused as we use the term pipeline almost everywhere, for different purposes. The tutorial-style instructions mentioned earlier should help avoid this confusion.